react/components/molecules/button-toggle/button-toggle.js

```diff
@@ -16,7 +16,11 @@ export class ButtonToggle extends mix(PureComponent).with(IdentifiableMixin) {
             color: PropTypes.string,
             colorSecondary: PropTypes.string,
             value: PropTypes.bool,
-            orientation: PropTypes.string,

```

Comment:
onPress?

Reply:
I'm using a logic of callback, where it receives a callback and calls it on onPress

Reply:
I'll change to onPress